resources/assets/sass/app.scss

```diff
@@ -10,8 +10,21 @@ $plyr-control-color: $colorHighlight;
 $plyr-control-bg-hover: $colorHighlight;
 $plyr-volume-track-height: 6px;
 $plyr-control-spacing: 6px;
+
 @import "vendor/_plyr.scss";
 
+.plyr {
```

Comment:
Exactly :)

On Sun, Jun 12, 2016 at 6:33 PM, Daniele Ricci <notifications@github.com>
wrote:

> In resources/assets/sass/app.scss
> <https://github.com/phanan/koel/pull/356#discussion_r66719167>:
>
> >  @import "vendor/_plyr.scss";
> >
> > +.plyr {
>
> No problem, but did you mean site-footer/index.vue maybe?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/phanan/koel/pull/356/files/825b5eed351b3ea2bf74747b7e45f3edd8c8ab84#r66719167>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHrt0nP_o4ppoX1wNa0qsoRT5N8CZzRoks5qK-CKgaJpZM4Izlab>
> .
>